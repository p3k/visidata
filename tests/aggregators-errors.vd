sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/test.jsonl	o	
test	key1		type-date		@	
test	qty		type-int		#	
test	amt		type-float		%	
test	qty		aggregate-col	sum	+	
test	key1		key-col		!	
test			columns-sheet		C	
test_columns		キqty	select-row		s	
test_columns		キamt	select-row		s	
test_columns			aggregate-cols	min	g+	
test_columns			aggregate-cols	max	g+	
test_columns			quit-sheet		q	
test		numeric_binning	set-option	True		
test	key2		pivot		W	
test_pivot_key2			columns-sheet		C	
test_pivot_key2_columns	name		sort-asc		[	
test_pivot_key2_columns			quit-sheet		q	
test_pivot_key2	key1		sort-asc		[	
