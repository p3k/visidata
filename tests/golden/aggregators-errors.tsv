key1	amt_max_bar	amt_max_baz	amt_max_foo	amt_min_bar	amt_min_baz	amt_min_foo	qty_max_bar	qty_max_baz	qty_max_foo	qty_min_bar	qty_min_baz	qty_min_foo	qty_sum_bar	qty_sum_baz	qty_sum_foo
#ERR		43.20			3.30			32			4		0	36	0
2016-01-01 - 2016-12-07									1			1	0	0	1
2016-12-07 - 2017-11-13													0	0	0
2017-11-13 - 2018-10-20	0.30	0.01	0.01	0.30	0.01	0.01	16	256		16	256		16	256	0
